npeps/npep-308.md

```diff
@@ -0,0 +1,127 @@
+# NPEP-308: Extending NetworkPolicy for Multi-Cluster support via `networking.k8s.io/multicluster-name` label
+
+* Issue: [#308](https://github.com/kubernetes-sigs/network-policy-api/issues/308)
+* Status: Provisional
+
+## TLDR
+
+This proposal introduces a standardized method for defining multi-cluster
+`NetworkPolicies` by extending the existing Kubernetes `NetworkPolicy` API. This
+is achieved by giving a new meaning to the well-defined label,
+`networking.k8s.io/multicluster-name`, in the `podSelector` and
+`namespaceSelector`, enabling policies to target resources across different
```

Comment:
Not `podSelector`. Just `namespaceSelector`. If your peer has no `namespaceSelector` then it only selects pods in the same namespace as the policy, so multicluster is irrelevant. If it has both a `namespaceSelector` and a `podSelector`, then the `podSelector` only selects pods in the namespaces selected by the `namespaceSelector`, so either you have a multi-cluster `namespaceSelector`, or you don't.

Reply:
yep I also got confused with why proposal had it for podselector level as well